client/src/js/services/grid/GridGrouping.js

```diff
@@ -148,6 +148,13 @@ function GridGroupingService(GridAggregators, uiGridGroupingConstants, Session,
     $timeout(api.treeBase.expandAllRows, 0, false);
   }
 
+  function unfoldGroup(row) {
+    var api = this.gridApi;
+    $timeout(function () {
```

Comment:
I get the feeling `$timeout` has a negative impact on performance of this grid and we have a lot of them.